src/public/modules/forms/admin/controllers/edit-start-page-modal.client.controller.js

```diff
@@ -43,8 +43,14 @@ function EditStartPageController(
 
   vm.saveStartPage = function (isValid) {
     vm.hasClickedSave = true
+    const logoState = vm.myform.startPage.logo.state
```

Comment:
would it be better if we cloned the startPage before resetting the logo? So the parameter to updateStartPage is the cloned object, but will not affect the "real" start page in case of failure

Reply:
good point! will change